client/actions/stats/getCategoryCompanyGender.js

```diff
@@ -0,0 +1,24 @@
+import request from 'superagent'
+import {showError} from '../index'
+
+export const GET_COMPANY_GENDER = 'GET_COMPANY_GENDER'
+
+const getCompanyGender = (stats) => {
```

Comment:
rename to `receiveCompanyGenderStats` to be consistent with other actions

Reply:
done